### PR TITLE
Enable one creditor to use multiple UCIs

### PR DESCRIPTION
**Pull request by igorkosta**
It will need some more polishing, because of the changes @schorsch did on cleaning up the $ref mess.

### CODE REVIEW

File: schema/v1.0/creditory_identity.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          "description" : "Results per page. Default is 10, max is 100",
+          "type"        : "integer"
+        },
+        "sort" : {
+          "title" : "Sort",
+          "enum"  : [
+            "ASC",
+            "DESC"
+          ],
+          "description" : "Sort the results in ASC or DESC",
```
Comment (razielgn) [inline schema/v1.0/creditory_identity.json]: On which property is it possible to sort?

File: schema/v1.0/sepa_mandate.json [hunk 2]
```diff
@@ -17,8 +17,10 @@
       "description" : "Customer for which this mandate is valid.",
       "type"        : "string"
     },
-    "user_id" : {
-      "$ref" : "./base_types/base_types.json#definitions/user_id"
+    "creditor_identity" : {
+      "description" : "The unique identifier of the creditor who created the mandate",
```
Comment (razielgn) [inline schema/v1.0/sepa_mandate.json]: Maybe instead of `unique identifier`, saying `label` would be more helpful?

Comment (igorkosta) [inline schema/v1.0/sepa_mandate.json]: let me change it

File: schema/v1.0/creditory_identity.json (hunk 1 above)
Comment (igorkosta) [inline schema/v1.0/creditory_identity.json]: There were some properties but they went missing in the big evening brain storm yesterday :)

**Comment by razielgn** [comment]
:+1: :ship:

**Comment by schorsch** [comment]
The label seems to be a unique marker, so why not use that as id.
Or we remove the id and only use the label as unique key, as i dont see the id beeing used elsewhere.

Depending on what we decide to use or kick, the related field in sepa_mandate should have the fieldname appended e.g creditor_identifier_label / _id

**Comment by razielgn** [comment]
Makes sense. Yes we can ditch `label`, have the `id` to replace it and `creditor_identifier_id` on the `sepa_mandate` schema.

**Comment by razielgn** [comment]
Latest changes look good! Let's merge it! :)

**Pull request merged**